src/dodal/devices/areadetector/plugins/MXSC.py

```diff
@@ -19,16 +34,37 @@ class PinTipDetect(Device):
 
     triggered_tip: Signal = Component(Signal, kind=Kind.hinted, value=INVALID_POSITION)
     validity_timeout: Signal = Component(Signal, value=5)
+    settle_time_s: Signal = Component(Signal, value=0.5)
+
+    tip_positions: List[Tuple] = []
```

Comment:
It looks like we don't need to define `tip_positions` here if it's also defined in `trigger()`